operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServerUpIteratorStepTest.java

```diff
@@ -0,0 +1,255 @@
+import oracle.kubernetes.operator.work.Step;
+import oracle.kubernetes.operator.work.Step.StepAndPacket;
+import oracle.kubernetes.operator.work.TerminalStep;
+import oracle.kubernetes.utils.TestUtils;
+import oracle.kubernetes.weblogic.domain.ClusterConfigurator;
+import oracle.kubernetes.weblogic.domain.DomainConfigurator;
+import oracle.kubernetes.weblogic.domain.DomainConfiguratorFactory;
+import oracle.kubernetes.weblogic.domain.model.Domain;
+import oracle.kubernetes.weblogic.domain.model.DomainSpec;
+import org.jetbrains.annotations.NotNull;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static oracle.kubernetes.operator.steps.ManagedServerUpIteratorStepTest.TestStepFactory.getServers;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+public class ManagedServerUpIteratorStepTest {
+
+  private static final String DOMAIN = "domain";
+  private static final String NS = "namespace";
+  private static final String UID = "uid1";
+  private static final String ADMIN = "asName";
+  private static final String CLUSTER = "cluster1";
+  private final Domain domain = createDomain();
+  private final DomainConfigurator configurator = DomainConfiguratorFactory.forDomain(domain);
+  private WlsDomainConfigSupport configSupport = new WlsDomainConfigSupport(DOMAIN);
+
+  private Step nextStep = new TerminalStep();
+  private FiberTestSupport testSupport = new FiberTestSupport();
+  private List<Memento> mementos = new ArrayList<>();
+  private DomainPresenceInfo domainPresenceInfo = createDomainPresenceInfo();
+  private TestUtils.ConsoleHandlerMemento consoleHandlerMemento;
+
+  private DomainPresenceInfo createDomainPresenceInfo() {
+    return new DomainPresenceInfo(domain);
+  }
+
+  private Domain createDomain() {
+    return new Domain().withMetadata(createMetaData()).withSpec(createDomainSpec());
+  }
+
+  private V1ObjectMeta createMetaData() {
+    return new V1ObjectMeta().namespace(NS);
+  }
+
+  private DomainSpec createDomainSpec() {
+    return new DomainSpec().withDomainUid(UID).withReplicas(1);
+  }
+
+  /**
+   * Setup env for tests.
+   * @throws NoSuchFieldException if TestStepFactory fails to install
+   */
+  @Before
+  public void setUp() throws NoSuchFieldException {
+    mementos.add(consoleHandlerMemento = TestUtils.silenceOperatorLogger());
+    mementos.add(TestStepFactory.install());
+    testSupport.addDomainPresenceInfo(domainPresenceInfo);
+  }
+
+  /**
+   * Cleanup env after tests.
+   * @throws Exception if test support failed
+   */
+  @After
+  public void tearDown() throws Exception {
+    for (Memento memento : mementos) {
+      memento.revert();
+    }
+
+    testSupport.throwOnCompletionFailure();
+  }
+
+  @Test
+  public void verifyThat_withConcurrencyOf1_bothClusteredServersShouldStartSequentially() {
```

Comment:
You don't need to prefix your test names with "verifyThat_" since all tests do verification. Just make it an assertion on the behaviors, such as:

withConcurrencyOf1_clusteredServersStartSequentially

Reply:
thanks. updated test names.